### ISSUE TITLE
Charaktererstellung

### ISSUE THREAD

**Issue by BroderPeters** [jmattheis/UnknownProject#35]
Hier soll erfasst werden welche Attribute und Eigenschaften zum Spielbeginn mit dem Charakter erstellt werden. Bspw aussehen(körperteile, accessoires), persönliche eigenschaften

**Comment by ColinWojciechowski** [comment]
Ich habe es jetzt so gelöst, dass es eine Klasse "Specieses" gibt [Name muss noch verbessert werden],
von der jede Rasse (Mensch, Ork Elf) erbt. Ausserdem hat jedes Objekt dieser Klasse noch eine Klasse "Job", mit der die Attribute verbessert werden. Folgende Attribute sind mit spontan in den Sinn gekommen, daran können wir gerne noch arbeiten:

HP -> Lebenspunkte
MP -> Manapunkte
Strength -> Stärke
Defense -> Verteidigung
Dex -> Geschicklichkeit (bsp: Dieb=Chanche zu Stehlen hoch)
Init -> Initiative (Der mit dem höhsten Wert beginnt den Kampf)

**Comment by jmattheis** [comment]
@ColinWojciechowski les dir dazu mal #1 durch (:

**Comment by ColinWojciechowski** [comment]
Hab #1 gelesen, bin aber nicht 100% schlauern ^^ Also realtime im Sinne von Diablo?!

**Comment by jmattheis** [comment]
ja, @BroderPeters hatte vorgeschlagen ein Teammeeting zumachen, das wir die so den aktuellen stand sagen können, was hältst du davon?

**Comment by BroderPeters** [comment]
@ColinWojciechowski @jmattheis Was haltet ihr von Mittwochabend? Circa 20 Uhr.
@nicolasries Natürlich auch bei Interesse

**Comment by ColinWojciechowski** [comment]
Habe jmattheiss schon gesagt, dass es zeitlich bei mir nie so einfach ist. Morgen habe ich jedenfalls keine Zeit. 
Wollt ihr euch treffen, TS oder wie?

**Comment by jmattheis** [comment]
@ColinWojciechowski auf einem ts ja

**Comment by BroderPeters** [comment]
TS und wenn nötig teamviewer
